templates/wbs_deliverables_layout.html

```diff
@@ -1,1283 +1,18 @@
+<!DOCTYPE html>
```

Comment:
DOCTYPE should come first to avoid quirks mode. I took almost everything out of this wbs_deliverables_layout.html file so that it is more maintainable.